src/engine/renderer/glsl_source/fogEquation_fp.glsl

```diff
@@ -0,0 +1,34 @@
+/*
+===========================================================================
+Copyright (C) 1999-2005 Id Software, Inc.
+Copyright (C) 2006-2011 Robert Beckebans <trebor_7@users.sourceforge.net>
```

Comment:
You don't have to give this organization and this individual a copyright on your own code.
Also, they don't have any power to compel you to use that license, please use BSD.

Reply:
OK, thanks.